pkg/config/config.go

```diff
@@ -181,14 +182,19 @@ func init() {
 	// Process Agent
 	BindEnvAndSetDefault("process_agent_enabled", true) // this is to support the transition to the new config file
 
-	// Log Agent
-	Datadog.SetDefault("log_open_files_limit", 100)
-	Datadog.SetDefault("logging_frequency", int64(20))
-	BindEnvAndSetDefault("log_enabled", false)
+	// Logs Agent
+	BindEnvAndSetDefault("logs_enabled", false)
+	BindEnvAndSetDefault("log_enabled", false) // deprecated, use logs_enabled instead
 	BindEnvAndSetDefault("logset", "")
-	BindEnvAndSetDefault("log_dd_url", "intake.logs.datadoghq.com")
-	BindEnvAndSetDefault("log_dd_port", 10516)
-	BindEnvAndSetDefault("run_path", defaultRunPath)
+
+	Datadog.SetDefault("logs_config.dd_url", "intake.logs.datadoghq.com")
+	Datadog.BindEnv("logs_config.dd_url", "DD_LOGS_CONFIG_DD_URL")
+
+	Datadog.SetDefault("logs_config.dd_port", 10516)
+	Datadog.BindEnv("logs_config.dd_port", "DD_LOGS_CONFIG_DD_PORT")
```

Comment:
this works 👍 

We can discuss (later) if we'd want to extend the substitution (`.` to `_` in the env var name) to all nested config keys (using viper's `SetEnvKeyReplacer(string.Replacer)`). But for now this works, let's keep this.

Reply:
:100:  with @olivielpeau 

It will help us to organize and standardize our configuration then avoid using something like:
```go
Datadog.BindEnv("collect_kubernetes_events")
Datadog.BindEnv("kubernetes_collect_service_tags")
```